rebar.config

```diff
@@ -1,23 +1,20 @@
-{require_otp_vsn, "R16|17|18"}.
+{minimum_otp_vsn, "R16"}.
 
-{erl_opts, [warnings_as_errors, {parse_transform, lager_transform}, debug_info, warn_untyped_record]}.
+{erl_opts, [warnings_as_errors,
+            {parse_transform, lager_transform},
+            debug_info,
+            warn_untyped_record]}.
 
-{eunit_opts, [verbose]}.
-{cover_enabled, true}.
+{deps, [getopt, lager]}.
```

Comment:
Versioned dependencies would be nice here to make it clear which versions we require and prevent accidental upgrades.